static/retirement/v0api/index.md

```diff
@@ -0,0 +1,225 @@
+
+In order to get at similar data, you could still use our Search API:
+
+```txt
+https://api.beta.ons.gov.uk/v1/search?content_type=dataset_landing_page&q={dataset_id}
+```
+
+See the [search response mapping](#search-response-mapping) for the overall response mapping and [search item data mapping](#search-item-data-mapping) for the mappings for individual items.
+
+### /dataset/{dataset\_id}/timeseries
+
+| Version | URI                                                                            |
+|---------|--------------------------------------------------------------------------------|
+| Old     | `{domain}/dataset/{dataset_id}/timeseries`                                     |
+| New     | `https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&q={dataset_id}` |
+
+See the [search response mapping](#search-response-mapping) for the overall response mapping and [search item data mapping](#search-item-data-mapping) for the mappings for individual items.
+
+### /dataset/{dataset\_id}/timeseries/{timeseries\_id} and /dataset/{dataset\_id}/timeseries/{timeseries\_id}/data
+
+| Version | URI                                                                               |
+|---------|-----------------------------------------------------------------------------------|
+| Old     | `{domain}/dataset/{dataset_id}/timeseries/{timeseries_id}`                        |
+| New     | `https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&q={timeseries_id}` |
+
+`/dataset/{dataset_id}/timeseries/{timeseries_id}` is a subset of what is retrieved via `/dataset/{dataset_id}/timeseries/{timeseries_id}/data`.
+
+After getting the response from our search service, you can get the `uri` parameter of the `item` that matches your `timeseries_id` to it's `cdid`. Using that `uri` you can then submit that to our data endpoint, for example:
+
+`https://api.beta.ons.gov.uk/v1/data?uri=/economy/nationalaccounts/uksectoraccounts/timeseries/mm23/capstk`
+
+These uris are currently evergreen and so won't change between editions.
+
+### /timeseries
+
+| Version | URI                                                              |
+|---------|------------------------------------------------------------------|
+| Old     | `{domain}/timeseries`                                            |
+| New     | `https://api.beta.ons.gov.uk/v1/search?&content_type=timeseries` |
+
+Parameters:
+
+| Old parameter | New parameter |
+|---------------|---------------|
+| start         | offset        |
+| limit         | limit         |
+
+You will need to add a new parameter to the request to just return the `dataset` content type:
+
+`content_type=dataset`
+
+See the [search response mapping](#search-response-mapping) for the overall response mapping and [search item data mapping](#search-item-data-mapping) for the mappings for individual items.
+
+### /timeseries/{timeseries\_id}
```

Comment:
is the "\" supposed to be in there?

Reply:
where?

Reply:
Please ignore

Reply:
https://github.github.com/gfm/#backslash-escapes 
I believe so - to avoid the underscore being treated as markdown

Reply:
ironically github showed this just as "" to me. :D